src/content-helper/excerpt-generator/provider.ts

```diff
@@ -35,28 +45,18 @@ export class ExcerptGeneratorProvider {
 	 * @return {Promise<string>} The generated excerpt.
 	 */
 	public async generateExcerpt( title: string, content: string ): Promise<string> {
-		let response;
-		try {
-			response = await apiFetch<ExcerptGeneratorApiResponse>( {
-				method: 'POST',
-				path: addQueryArgs( '/wp-parsely/v1/content-suggestions/suggest-brief', {
-					title,
-				} ),
-				data: {
-					content,
-				},
-			} );
-		} catch ( wpError: any ) { // eslint-disable-line @typescript-eslint/no-explicit-any
-			return Promise.reject( new ContentHelperError( wpError.message, wpError.code ) );
-		}
-
-		if ( response?.error ) {
-			return Promise.reject( new ContentHelperError(
-				response.error.message,
-				ContentHelperErrorCode.ParselyApiResponseContainsError
-			) );
+		if ( '' === title ) {
+			title = 'Untitled';
```

Comment:
Should we make that string translatable?

Reply:
I don't think it's necessary, since this string will be sent to the LLM and will not be present to the user.

Reply:
OK, feel free to merge as-is. I wonder if this might be of any significance if we start supporting different languages.

Reply:
Yeah, might be! However I think the content language would have more impact than the actual WordPress language, eg, if I have my WP install in French, but the content is in English, it wouldn't make sense to send the translated string.

I don't think it's a trivial problem to solve, and in this specific scenario I don't think the gains are actually worth the trouble.

Reply:
Yes, totally agree. Thank you for your work! 🙂